python/graphstorm/dataloading/dataloading.py

```diff
@@ -1785,51 +1788,45 @@ def __len__(self):
                    self.unlabeled_dataloader.expected_idxs)
 
 class GSgnnRealtimeInferNodeDataLoader(GSgnnNodeDataLoaderBase):
-    """ Mini-batch dataloader for real-time node infernce task
+    """ Mini-batch dataloader for real-time node inference task
 
     .. versionadded:: 0.5
-        Add `GSgnnRealtimeInferNodeDataLoader` class to support node-level dataloader used for
+        Add `GSgnnRealtimeInferNodeDataLoader` class to support node-level dataloader for
         real-time inference.
 
-    This dataloader wraps DGL's DataLoader for real-time inference in GraphStorm. It will use DGL's
-    `MultiLayerFullNeighborSampler` samper to extract all nodes from the given subgraph because
-    during real-time inference, it is unclear if data providers have done fanout sampling or not
-    when building the subgraph payload.
-    
-    This class extends from the ``GSgnnNodeDataLoaderBase`` class so that it will be compatible
-    with other dataloaders.
+    This dataloader extends from the ``GSgnnNodeDataLoaderBase`` class for real-time inference.
+    It will use DGL's `MultiLayerFullNeighborSampler` sampler to extract all nodes from the given
+    subgraph during real-time inference. Graph sampling can be achieved before invoking the
+    real-time inference endpoint when building the subgraph payload.
 
     Parameters
     -----------
-    dataset: DGLGraph
+    g: DGLGraph
         A DGLGraph instance.
     target_idx : dict of Tensors
         The target node indexes for prediction.
     num_layers: int
-        The number of layers dataloader will use as the number of hops for sampling. Default is 1.
+        The number of layers dataloader will use as the number of GNN layers. Default is 1.
     """
     def __init__(self,
-                 dataset,
+                 g,
                  target_idx,
                  num_layers=1):
-        assert isinstance(dataset, dgl.DGLGraph), ('The \"dataset\" should be a DGLGraph ' \
-            f'instance, but got {dataset}.')
-        assert isinstance(target_idx, dict), ('The \"target_idx\" should be a dictionary, ' \
-            f'but got {target_idx}.')
+        assert isinstance(g, dgl.DGLGraph), ('The argument \"g\" should be a DGLGraph ' \
+            f'instance, but got {g}.')
+        assert isinstance(target_idx, dict), ('The argument \"target_idx\" should be a ' \
+            f'dictionary, but got {target_idx}.')
         for ntype in target_idx:
-            assert ntype in dataset.ntypes, (f'node type {ntype} does not exist in the graph.')
-
-        # set dummy label fields using "label" string
-        label_field = "label"
+            assert ntype in g.ntypes, (f'node type {ntype} does not exist in the graph.')
 
-        super().__init__(dataset,
+        super().__init__(g,
                          target_idx,
                          fanout=[-1],
-                         label_field=label_field,
+                         label_field="label",
```

Comment:
why this is a constant value?

Reply:
The base class `GSgnnNodeDataLoaderBase` asks for a valid l`abel_field`. So, here give it a dummy name. Added some comments to explain.